src/commcare_cloud/terraform/modules/openvpn/variables.tf

```diff
@@ -3,7 +3,7 @@ data "aws_ami" "openvpn_image" {
   most_recent = true
   filter {
     name   = "name"
-    values = ["OpenVPN Access Server 2.5.0-3b5882c4-551b-43fa-acfe-7f5cdb896ff1-ami-548e4429.4"]
+    values = ["OpenVPN Access Server 2.6.1-fe8020db-5343-4c43-9e65-5ed4a825c931-ami-0f5d312e085235ed4.4"]
```

Comment:
will this trigger a change on prod as well?

Reply:
It _would_, but in this PR I also override this in [`production/terraform.yml`](https://github.com/dimagi/commcare-cloud/pull/2685/files#diff-aca45def687ae46ebf00815b019dfef8) as well as `staging/terraform.yml`, to their current value.

The original query was no longer working, possibly because of the change of region. These updates combined made it run smoothly on india, while also not resulting in any changes to production or staging